packages/studio-base/src/players/types.ts

```diff
@@ -116,6 +116,14 @@ export type PlayerState = {
   // See `const PlayerCapabilities` for more details.
   capabilities: typeof PlayerCapabilities[keyof typeof PlayerCapabilities][];
 
+  /**
+   * Identifies the semantics of the data being played back, such as which topics or parameters are
+   * semantically meaningful or normalization conventions to use. This typically maps to a shorthand
+   * identifier for a robotics framework such as "ros1", "ros2", or "ulog". See the MCAP profiles
+   * concept at <https://github.com/foxglove/mcap/blob/main/docs/specification/appendix.md#well-known-profiles>.
+   */
+  profile: string;
```

Comment:
Make this optional? I have a slight preference for that over empty strings.

```suggestion
  profile?: string;
```